lib/Data.js

```diff
@@ -55,13 +66,13 @@ export class Suite {
 //TODO: good idea?
 //Mocha just passes the exception thrown by the assertion library
 /*
-export class Error {
-    constructor(name, message, stack, expected, actual){
-        this.name = name;
-        this.message = message;
-        this.stack = stack;
-        this.expected = expected;
-        this.actual = actual;
-    }
-}
-/**/
+ export class Error {
+ constructor(name, message, stack, expected, actual){
+ this.name = name;
+ this.message = message;
+ this.stack = stack;
+ this.expected = expected;
+ this.actual = actual;
+ }
+ }
```

Comment:
Seems fine to ignore, we can address that once we get to implementing `test.errors`.